scripts/prettier.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
While its good to have a seperate file for this , we should incorporate this into our `format.sh` and `lint.sh` scripts too.